tools/aws_lab_setup/README.md

```diff
@@ -54,82 +76,87 @@ To set up the lab for Ansible training, follow these steps.
-
-10. Clone the lightbulb repo:
-
-        git clone https://github.com/ansible/lightbulb.git
-        cd lightbulb/tools/aws_lab_setup
-
-11. Define the following variables, either in a file passed in using `-e @extra_vars.yml` or directly in a `vars` section in `aws_lab_setup\infra-aws.yml`:
-
-      ```yaml
-      ec2_key_name: username                # SSH key in AWS to put in all the instances
-      ec2_region: us-west-1                 # region where the nodes will live
-      ec2_name_prefix: TRAINING-LAB         # name prefix for all the VMs
-      ec2_vpc_id: vpc-1234aaaa              # EC2 VPC ID in your region
-      ec2_vpc_subnet_id: subnet-5678bbbb    # EC2 subnet ID in your VPC
-      admin_password: changeme123           # Set this to something better if you'd like. Defaults to 'LearnAnsible[two digit month][two digit year]', e.g., LearnAnsible0416
-      ## Optional Variables
-      email: no                             # Set this if you wish to disable email
-      sendgrid_user: username               # username for the Sendgrid module.  Not required if "email: no" is set
-      sendgrid_pass: 'passwordgoeshere'     # sendgrid accound password.  Not required if "email: no" is set
-      sendgrid_api_key: 'APIkey'            # Instead of username and password, you may use an API key. Don't define both. Not required if "email: no" is set
-      instructor_email: 'Ansible Instructor <helloworld@acme.com>'  # address you want the emails to arrive from. Not required if "email: no" is set
-      ```
-
-12. Create a `users.yml` by copying `sample-users.yml` and adding all your students:
-
-    __(email)__
-    ```yaml
-    users:
-      - name: Bod Barker
-        username: bbarker
-        email: bbarker@acme.com
-
-      - name: Jane Smith
-        username: jsmith
-        email: jsmith@acme.com
-    ```
-
-    __(no email)__
-    ```yaml
-    users:
-      - name: Student01
-        username: student01
-        email: instructor@acme.com
-
-      - name: Student02
-        username: student02
-        email: instructor@acme.com
-    ```
-    **(no email) NOTE:**  If using generic users, you can generate the corresponding
+### Setup (per workshop)
+
+1. Define the following variables in a file passed in using `-e @extra_vars.yml`
+
+```yml
+ec2_key_name: username                # SSH key in AWS to put in all the instances
+ec2_region: us-east-1                 # region where the nodes will live
+ec2_az: us-east-1a                    # availability zone
+ec2_name_prefix: TRAINING-LAB         # name prefix for all the VMs
+admin_password: ansible
+## Optional Variables
+email: no                             # Set this if you wish to disable email
+localsecurity: false                   # skips firewalld installation and SE Linux when turned to false
+```
+
+For an example, look at [sample-vars.yml](sample-vars.yml) for a list of all the knobs you can control.  For example you can use pre-existing AWS VPCs you already created.
```

Comment:
"For example"... "For example" -> Delete the first one.

Reply:
fixed in commit: https://github.com/network-automation/lightbulb/commit/aa78f130ba60c0ac3f71e7a8b85fdb1cfc1969d9